webScraping/cfeScrapper.js

```diff
@@ -9,23 +9,23 @@
  * any table can be collected and saved in some file for later use.
  */
 
-const puppeteer = require('puppeteer');
-const ScrapPage = require('../scrapperFunctions/index');
-const URLPage = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
-const idInput = '#descProc';
-const idButton = '#buscar';
-const waitingTime = 2000;
-// const route = './assets/Data-From-Table.json';
-const nextPageBtn = 'div.row a.k-link span.k-i-arrow-e';
-const tableSelector = 'table.k-selectable';
-const rowSelector = '#totProc';
+const PUPPETEER = require('puppeteer');
+const SCRAP_PAGE = require('../scrapperFunctions/index');
+const URL_PAGE = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+const ID_INPUT = '#descProc';
+const ID_BUTTON = '#buscar';
+const WAITING_TIME = 2000;
+// const ROUTE = './assets/Data-From-Table.json';
```

Comment:
I have deleted this line a lot of times, but you always put it again, delete it please